run_tests.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+# Default values
+NETWORK="regtest"
+LOG_LEVEL="none"
+LONG_TESTS=0
+BACKTRACE=0
+
+# Parse command-line arguments
+while getopts "l:b:t:L" opt; do
+  case $opt in
+    l)
+    if [ "$OPTARG" == "debug" ] || [ "$OPTARG" == "info" ] || [ "$OPTARG" == "none" ]; then
+        LOG_LEVEL=$OPTARG
+    else
+        echo "Invalid log level option. Use 'info', 'debug', or 'none'."
+        exit 1
+    fi
+    ;;
+    b)
+      BACKTRACE=$OPTARG
+      ;;
+    t)
+      TEST_NAME="$OPTARG"
+      ;;
+    L)
+      LONG_TESTS=1
```

Comment:
what is long test ?

Reply:
Long tests are the tests I added. When running in signet they take a long time to run, as I am building the transaction and then checking it.